src/index.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
```

Comment:
Migrate `src/index.css` to `styled-component` using `globalStyles`. can you tell me why?

Reply:
Maybe the code will be cleaner because the styles are made in a separate component.